setu-planning/v1.0/PlanningAssignment/PlanningAssignment_Example_v1.0.json

```
@@ -61,13 +61,13 @@
         "schemeAgencyId": "Customer"
       },
       "planningPeriod": "Monday",
-      "startTime": "07:00:00",
-      "endTime": "16:00:00",
+      "startTime": "07:00:00Z",
+      "endTime": "16:00:00Z",
```

Alternatively, to make sure it's a Dutch example, use +01:00 or +02:00
```suggestion
      "startTime": "07:00:00+01:00",
      "endTime": "16:00:00+01:00",
```